projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-grid.component.ts

```diff
@@ -676,29 +663,13 @@ export class IgxHierarchicalGridComponent extends IgxHierarchicalGridBaseCompone
 
     private hg_verticalScrollHandler(event) {
         this.scrollTop = event.target.scrollTop;
-        this.hideOverlays();
     }
 
-    public onContainerScroll() {
-        this.hideOverlays();
```

Comment:
This should not be removed. In scenarios where the child grid has larger width than the parent a scrollbar is rendered. Currently  when scrolling via that scrollbar overlays will not close, as a result they may go outside the scope of their parent container:

![image](https://user-images.githubusercontent.com/10397980/55711457-59cd5800-59f5-11e9-9859-559f13a8fe68.png)